### ISSUE TITLE
Arturia Beatstep pads do not send any commands

### ISSUE THREAD

**Issue by leachim6** [git-moss/DrivenByMoss#259]
Hello, I am having an issue in which the pads on my Arturia Beatstep send no interractive commands inside the host DAW.

I have tried both in Reaper and Bitwig Studio
###  **The Setup:**
Cockos Reaper 6.35
Bitwig Studio 4.0.1
Windows 10 Pro Build 19043

I have also purged all traces of DrivenByMoss from my reaper resource path and bitwig extensions directory between each test.

I have tried the Bitwig and reaper versions of the latest binaries (13.6.0) as well as building the latest HEAD fd28fb42c05227ffd448c46b6810a7442dfd65d3 from source for Bitwig (I was not able to find source code for the reaper version, is that available?)

I have loaded the configuration from resources/Beatstep/DrivenByMoss.beatstep onto the device using MIDI Control Center.
I have repeatedly reset the device to factory settings and transferred the moss configuration into multiple "slots" just to rule out an inconsistency there.

I have ensured the global midi channel is set to 3, although I have tried the other 15 channels to make sure.

###  **The Behavior:**

All of the pad LEDs on the Beatstep while at rest and pressing shift correspond to their proper functions. However, pressing any of the pads (besides the shift key) are ineffectual. The output in both reaper and bitwig studio from the debug log is as follows:

![image](https://user-images.githubusercontent.com/4377/131493422-4839a0b1-1be6-46d5-a3f9-91b481c03a73.png)
```Midi Note {on/off} {#} should be handled in framework...```

It appears to me that the pads are not sending the proper midi messages. However, I was not able to determine what the proper midi messages should be. 

The MIDI Control Center indicates that the pads are all set to midi notes starting at C1 on pad 9 chromatically through pad 16, then continuing in a Z shape from pad 16 to pad 8 ending on D#2 

![image](https://user-images.githubusercontent.com/4377/131494139-b9068054-a3a0-4e92-ab20-68cf67973041.png)

Should these pads be midi CCs or are these notes correct?

Please Advise.

Thank you for your time 👍

**Comment by git-moss** [comment]
Yes, notes are correct. The only thing I can think of that you accidently transposed them (but I guess this should be reset when reset the device).

From the manual: "The big knob is used to move the play cursor in the Arrange view. Warning: Don’t use it in combination with the Shift button or you will (still) transpose the midi notes sent by the pads, which will make the pads
act funny."

If this does not help, please check with a MIDI monitor what notes/commands are actually sent by the device.

**Comment by leachim6** [comment]
Hi Moss,

Thanks for the quick reply! Can you confirm what the correct notes should be for the pads so I can double check?

**Comment by leachim6** [comment]
![image](https://user-images.githubusercontent.com/4377/131501962-60b282c7-6ee5-41ab-8a97-764745f28f85.png)

Here is the behavior I'm seeing when for example, attempting to disarm a track for recording by selecting the track, holding shift and pressing pad 10

**Comment by git-moss** [comment]
Notes need to start with C1 (=36), which is shown correctly in your first picture but the Reaper screenshot shows that you are receiving notes below 36. Therefore, try to use Shift+Big Encoder to transpose them up again.

While testing I noticed several issues which I fixed:
  * Play button did not work. -> will be fixed in the next update
  * The template had set the wrong global MIDI channel (1 instead of 3). -> you noticed yourself
  * The manual missed the command tables for all modes. -> it is already checked in if you want to take a look.
